server/src/test/java/org/elasticsearch/common/util/concurrent/TaskExecutionTimeTrackingEsThreadPoolExecutorTests.java

```diff
@@ -41,16 +41,32 @@ public void testExecutionEWMACalculation() throws Exception {
         logger.info("--> executor: {}", executor);
 
         assertThat((long) executor.getTaskExecutionEWMA(), equalTo(0L));
+        assertThat(executor.getTotalTaskExecutionTime(), equalTo(0L));
         executeTask(executor, 1);
-        assertBusy(() -> { assertThat((long) executor.getTaskExecutionEWMA(), equalTo(30L)); });
+        assertBusy(() -> {
```

Comment:
nit: could we add a comment to note that `fastWrapper` makes each execution look like it took 100ns?